datalad_next/create_sibling_webdav.py

```diff
@@ -470,32 +543,29 @@ def _create_git_sibling(
         # e.g., it is not uncommon for credentials to be named after URLs
         remote_url += f'&dlacredential={urlquote(credential_name)}'
 
-    yield from ds.siblings(
-        # TODO set vs add, consider `existing`
-        action="add",
+    return ds.siblings(
```

Comment:
The kinda assumed that `siblings` will never give more than one result. I don't think this is a robust setup.

Reply:
done